enclave/core/sgx/cpuid.c

```diff
@@ -10,6 +10,7 @@
 #include "platform_t.h"
 
 static uint32_t _cpuid_table[OE_CPUID_LEAF_COUNT][OE_CPUID_REG_COUNT];
+extern const uint32_t supported_cpuid_leaves[OE_CPUID_LEAF_COUNT];
```

Comment:
Is this extra declaration necessary? This file already includes openenclave/internal/cpuid.h which declares the same.

Reply:
Oh, yes, I can remove the extern declaration in the C files. This reply is also suitable for the other comments below and related to this. I indeed firstly added the declarations in the C files and did some tests. And then refined the `oe_is_emulated_cpuid_leaf` function changing the cpuid.h, but forgot to remove the declarations in C files.  Thanks for catching this.